### PR TITLE
Filled out README and added docs directory

### PR DESCRIPTION
**Pull request by drpatelh**
This addresses https://github.com/nf-core/configs/issues/3

Also added `docs` directory to upload documentation specific to custom config files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+
+[Fork](https://help.github.com/articles/fork-a-repo/) the `nf-core/configs` repository to your own GitHub account. Within the local clone of your fork add the custom config file to the [`conf/`](https://github.com/nf-core/configs/tree/master/conf) directory, and the documentation file to the [`docs/`](https://github.com/nf-core/configs/tree/master/docs) directory. You will also need to edit and add your custom profile to the [`nfcore_custom.config`](https://github.com/nf-core/configs/blob/master/nfcore_custom.config) file in the top-level directory of the clone.
+
+Commit and push these changes to your local clone on GitHub, and then [create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) on the `nf-core/configs` GitHub repo with the appropriate information.
+
+## Help
+
+We will be notified automatically when you have created your pull request, and providing that everything adheres to `nf-core` guidelines we will endeavour to approve your pull request as soon as possible.
+
+If you have any questions or issues please send us a message on [`Slack`](https://nf-core-invite.herokuapp.com/).
```
Comment (ewels) [inline README.md]:
```suggestion
If you have any questions or issues please send us a message on [Slack](https://nf-core-invite.herokuapp.com/).
```

Comment (drpatelh) [reply README.md]: Fixed with https://github.com/nf-core/configs/pull/7/commits/19526e45e8d50073503c93e11608f0fc8e300992

File: docs/crick.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# nf-core/configs: Crick (CAMP HPC) Configuration
+
+This pipeline has been successfully configured for use on the CAMP HPC cluster at the [The Francis Crick Institute](https://www.crick.ac.uk/).
+
+To use, run the pipeline with `-profile crick`. This will download and launch the [`crick.config`](../conf/crick.config) which has been pre-configured with a setup suitable for the CAMP HPC cluster. Using this profile, Nextflow will download a temporary singularity image with all of the required software before execution of the pipeline.
+
+Before running the pipeline you will need to load Nextflow and Singularity using the environment module system on CAMP. You can do this by issuing the commands below:
+
+```
```
Comment (ewels) [inline docs/crick.md]: Get some syntax highlighting:
```suggestion
```bash
```

Comment (drpatelh) [reply docs/crick.md]: Fixed with https://github.com/nf-core/configs/pull/7/commits/08dce7453a5aa50cbdad0dac9ffe6e84ae29aa5f

**Comment by drpatelh** [comment]
giphy

**Comment by drpatelh** [comment]
giphy

File: README.md [hunk 3]
```diff
@@ -1,2 +1,64 @@
-# configs
-Config files used to define parameters specific to compute environments at different Institutions.
+<img src="docs/images/nf-core-logo.png" width="400">
+
+# [nf-core/configs](https://github.com/nf-core/configs)
+[![Build Status](https://travis-ci.org/nf-core/configs.svg?branch=master)](https://travis-ci.org/nf-core/configs)
```
Comment (ewels) [inline README.md]: There is no testing or code coverage for this repo, so no need for these badges..

**Pull request merged**